phenol-io/src/main/java/org/monarchinitiative/phenol/io/OntologyLoader.java

```diff
@@ -93,15 +93,16 @@ private static GraphDocument loadGraphDocument(InputStream inputStream) {
         } catch (Exception e) {
           throw new PhenolRuntimeException("Error loading JSON", e);
         }
+      } else {
+        try {
+          bufferedStream.reset();
+        } catch (Exception e) {
+          throw new PhenolRuntimeException("Error loading OBO/OWL", e);
+        }
       }
-
       logger.debug("Looks like a OBO/OWL file...");
-      try {
-        bufferedStream.reset();
-        return OboGraphDocumentLoader.loadObo(bufferedStream);
-      } catch (Exception e) {
-        throw new PhenolRuntimeException("Error loading OBO/OWL", e);
-      }
+      logger.error("OBO/OWL support was removed since 2.0.0");
+      throw new PhenolRuntimeException("OBO/OWL support was removed since 2.0.0, use JSON instead");
```

Comment:
I still think we might want to think about having a separate OWL/OBO module to support this if we need it in the future.

Reply:
OWL support could be a separate library that would extend phenol-core in an appropriate way. Currently, there are no applications that require OWL.

Reply:
Indeed. A `phenol-obo-owl` module with a reader and writer requiring `phenol-core` and `owl-api` would be all that was required. If it's not yet needed, it's not needed but would be simple enough to add.